README.md

```diff
@@ -1,2 +1,84 @@
-# azure-archetype-lib
-Repository with q.beyond designed Azure-archetypes, including policies
+# Archetype Library
+[![GitHub tag](https://img.shields.io/github/tag/qbeyond/terraform-azurerm-archetype-lib.svg)](https://registry.terraform.io/modules/qbeyond/terraform-azurerm-archetype-lib/provider/latest)
+[![License](https://img.shields.io/github/license/qbeyond/terraform-azurerm-archetype-lib.svg)](https://github.com/qbeyond/terraform-azurerm-archetype-lib/blob/main/LICENSE)
+
+----
+
+This [repository](https://github.com/qbeyond/terraform-azurerm-archetype-lib) is our central library of custom policy (set) definitions. Policies defined here will be usable by CAF archetypes.
+Over this Module, named [archetype-lib](https://registry.terraform.io/modules/qbeyond/archetype-lib/azurerm/latest), all q.beyond archetype-, policy-, policy set- and roledefinitions and policy assignments will be summarized in one folder with all the azurerm compatible definitions and assignments.
+That folder will containin the external and the q.beyond definitions and assignments. The planed usecase is to hand the output `merged_library` of this Module to the CAF-Module, as it can only handle one Folder as input containing definitions and assignements.
+
+## Root Repository Structure
+- archetypes is a folder just containing archetype definitions: no subfolders, the name serves as description
+- The main categories used by archetypes: policy_definition, role_definitions
+  - they each contain folders named by the topics that the policys or roles are for
```

Comment:
policies*

Reply:
Very german phrase, try: ... named by the corresponding topics of the policies and roles*